src/Microsoft.ML.Data/Transforms/TermEstimator.cs

```diff
@@ -3,15 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.StaticPipe;
 using Microsoft.ML.StaticPipe.Runtime;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms.CategoricalTransforms
 {
-    public sealed class TermEstimator : IEstimator<TermTransform>
+    public sealed class ValueToKeyMappingEstimator: IEstimator<TermTransform>
```

Comment:
Can we rename the file to match the class name?  Having the class `ValueToKeyMappingEstimator` in a file named `TermEstimator.cs` will make it impossible for people to navigate the file system to find the class.